template/scripts/pre-commit.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
```

Comment:
To test this, run `npm run build:complete` to create a new repo. Then in that repo make a simple change and try to commit. 

In that process, the questions should come up. 

Reviewing this personally, I am wondering as to opinions of burdon. I do have the script so that users can simply hit the `return` key and it will accept that as a `y`. But are there too many questions? Or, should there be only one question asking if the user reviewed the Auro Definition of Done document?